### ISSUE TITLE
Fix Request: AvoidCommonPoolForFutureAsync - supplyAsync false negative when using static import

### ISSUE THREAD

**Issue by hwestra** [jborgers/PMD-jPinpoint-rules#242]
*IA08 Observation: Future.supplyAsync is used for remote calls and it uses the common pool*
This rules does not fail due to static import in line 2 in the following example code:

```
import static java.util.concurrent.CompletableFuture.completedFuture;
import static java.util.concurrent.CompletableFuture.supplyAsync;

class SomeClass {
    public CompletableFuture<?> someMethod() {
        return supplyAsync(() -> {
            var result = someLogic();
            return completedFuture(result);
        });
    }
}
```

The rule for this indeed matches literally on the classname:
`PrimaryPrefix/Name[@Image='CompletableFuture.supplyAsync']/../../PrimarySuffix/Arguments/ArgumentList[count(Expression) = 1]`

Please extend the rule to also trigger when using static imports.
